0022-lnd-plan-of-attack.adoc

```diff
@@ -97,8 +97,11 @@ There is no plan to monitor the underlying blockchain for this.
 
 ==== Standard LN does not support payments without the other party knowing the pre-image
 
-There is an experimental feature called "hold-invoice" which we can utilize to design a symmetric protocol.
-As a consequence, users will have to compile LND with the feature flag "invoicesrpc" to get the desired functionality.
+HAN-HALight swaps can be achieved using standard Lightning.
```

Comment:
Yes, but actually @thomaseizinger suggested to still use the hodl invoice future for some reasons. The reasons are fair to me, I just don't remember them. It's actually written somewhere in the spike but I suggested @thomaseizinger to state it more explicitely, here would be a good place to do so.

Reply:
As the proposed protocol is symmetric it simplifies the development and results in the same user behaviour in both directions.

We can optimize it at a later stage.

Reply:
> As the proposed protocol is symmetric it simplifies the development and results in the same user behaviour in both directions.

There were more than that I believe.

Reply:
Sincere apologies @thomaseizinger, I did read that section (the first time I read the spike, before I really understood what was going on) - I did not remember that section and on subsequent readings did not see it somehow.  I finally get it now, for HAN-HALight the 'hold' period is essentially zero since Alice knows the secret but the protocol remains symmetrical.  Thanks for you patience.